platform/linux-generic/odp_stash.c

```diff
@@ -482,14 +597,14 @@ static inline int32_t stash_put(odp_stash_t st, const void *obj, int32_t num, od
 	obj_size = stash->obj_size;
 
 	if (obj_size == sizeof(uint64_t)) {
-		if (batch)
+		if (is_batch)
```

Comment:
Actually, perhaps this batch check could now be done once at the beginning of put/get and save correct functions (as function pointers are already used anyways) to variables and then use the variables, a bit less if-batch-else repetition.

(I seemingly whined about this also in #1636.)

Otherwise looks OK.

Reply:
Makes sense, fixed in V3.